Window grab API rework (#2379)
pygame-community/pygame-ce: docs/reST/ref/sdl2_video.rst

      depending on the window focus. To get if the mouse is currently restricted to this
      window, please use :attr:`mouse_grabbed`.

      .. seealso:: :attr:`mouse_grabbed`

      .. versionadded:: 2.4.0
   
   .. attribute:: grab_keyboard

      | :sl:`Get or set the window's keyboard grab mode`
      | :sg:`grab_keyboard -> bool`
      
      When this attribute is set to ``True``, the window will try to capture system 
      keyboard shortcuts like ``Alt+Tab`` or the ``Meta/Super`` key. 

      This attribute only set the "mode" of grab. The keyboard may be captured by
      another window depending on the window focus. To get if keyboard is currently 
      captured by this window, please use :attr:`keyboard_grabbed`.
      
      Note that not all system keyboard shortcuts can be captured by applications
      (one example is ``Ctrl+Alt+Del`` on Windows).

      When keyboard grab is enabled, pygame will continue to handle ``Alt+Tab`` when
      the window is full-screen to ensure the user is not trapped in your application.
      If you have a custom keyboard shortcut to exit fullscreen mode, you may suppress

>       this behavior with ``SDL_HINT_ALLOW_ALT_TAB_WHILE_GRABBED``.

Reviewer: ```suggestion
      this behavior with an environment variable, e.g.
       ``os.environ["SDL_ALLOW_ALT_TAB_WHILE_GRABBED"] = "0"``.
```

A small suggestion - just because it is not always obvious to our users that they need to remove the `_HINT` from SDL environment variable names to make them work in python/pygame-ce code.

      This attribute requires SDL 2.0.16+.

      .. seealso:: :attr:`keyboard_grabbed`

      .. versionadded:: 2.4.0
      
   .. attribute:: mouse_grabbed

      | :sl:`Get if the mouse cursor is confined to the window (**read-only**)`
      | :sg:`mouse_grabbed -> bool`

      Get if the mouse cursor is currently grabbed and confined to the window.

      Roughly equivalent to this expression:
         
      ::

         win.grab_mouse and (win is get_grabbed_window())
      
      .. seealso:: :attr:`grab_mouse`
      
      .. versionadded:: 2.4.0
   
   .. attribute:: keyboard_grabbed

>       | :sl:`Get if the keyboard shortcuts is captured by the window (**read-only**)`

Reviewer: ```suggestion
      | :sl:`Get if the keyboard shortcuts are captured by the window (**read-only**)`
```

Small grammar fix.

      | :sg:`keyboard_grabbed -> bool`

>       Get if the keyboard shortcuts is currently grabbed and captured by the window.

Reviewer: ```suggestion
      Get if the keyboard shortcuts are currently grabbed and captured by the window.
```
